packages/apify/src/apify.ts

```diff
@@ -305,7 +305,7 @@ export class Apify {
      *  Input overrides for the actor task. If it is an object, it will be stringified to
      *  JSON and its content type set to `application/json; charset=utf-8`.
      *  Provided input will be merged with actor task input.
-     * @param {object} [options]
+     * @param [options]
      * @throws {ApifyCallError} If the run did not succeed, e.g. if it failed or timed out.
      */
     async callTask(taskId: string, input?: Dictionary | Dictionary[], options: CallTaskOptions = {}): Promise<ActorRunWithOutput> {
```

Comment:
I know it's technically unrelated to this PR, but should we use the options from apify-client instead? CC @B4nan

Reply:
@vladfrangu should I do it here or you will update it? I see some options are not exactly the same (e.g. `memory` in actor/task start options, while here it's `memoryMbytes`. Same with `timeout` vs `timeoutSecs`, `waitSecs` vs `waitForFinish, `webhooks` are of the different type. Should I rename props too or extend it and keep the same names?

Reply:
Hello fellow night owl! 🦉
I'd say we all should have a call about this (especially the memory renames) and decide how we want to move forward. Does that sound good?

Reply:
:wave: 🦉 :smile:
Yep, I removed the unnecessary typecasting (your second comment). So I guess this point is the last thing to agree on before merging if @B4nan would not have any other comments. So let's have a call tomorrow.